plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityColor.java

```diff
@@ -280,7 +307,8 @@ public ListTag getAllowedColors() {
                 horseColors.addAll(Utilities.listTypes(Horse.Style.class));
                 yield horseColors;
             }
-            case SHEEP, WOLF, SHULKER -> Utilities.listTypes(DyeColor.class);
+            case WOLF -> Utilities.listTypes(Wolf.Variant.class);

```

Comment:
the allowed colors handling is a bit weird for cases like this, see eg how tropical_fish handles it